infrastructure/cli/commands/helm/history.ts

```diff
@@ -0,0 +1,45 @@
+import {helm, releaseName} from './index.js'
+import {namespace} from '../../lib/cluster.js'
+import {makeCommand} from '../../lib/command.js'
+import {shortDateTime} from '../../lib/misc.js'
+import {EnvironmentOptions} from '../../lib/options.js'
+import {printTable} from '../../lib/table.js'
+
+export const {command, description, builder} = makeCommand(
+  import.meta.url,
+  'List history of releases',
+)
+
+type Release = {
+  revision: number
+  updated: string
+  status: string
+  chart: string
+  app_version: string
+  description: string
+}
+
+export async function handler({environment}: EnvironmentOptions): Promise<void> {
+  const ns = namespace(environment)
+  const args = ['history', releaseName, '--namespace', ns, '--output', 'json']
+  const list = await helm(args, {output: 'object'}) as Release[]
+  const rows = list
+    .sort(({revision: revision1}, {revision: revision2}) => {
+      return revision2 - revision1
+    })
```

Comment:
I wonder if it's overkill to factor out this sort by revision function.

Shouldn't you also `parseInt(..., 10)` here as well or am I missing something and these sort functions are actually different and/or work on different types?

Reply:
the JSON responses from `helm list` and `helm history` return "revision" using different types! one is a string, the other a number. thanks helm